dist/css/style.css

```diff
@@ -52,19 +52,26 @@ img {
   margin: 0 1rem;
 }
 
-.list-item-img {
-  margin: 2rem;
-}
+.list-item-img {}
 
 .list-item-name {}
 
 .list-item-description {}
 
+.student-a {
```

Comment:
Is this a link?

Reply:
Yeah, I was just following the className pattern inside Student.jsx. I imagine we'll overhaul everything once we get to the design portion.

Reply:
Got it, thanks!